.github/workflows/csv-to-csvw.yml

```diff
@@ -52,52 +53,68 @@ jobs:
             file_name="${file_path##*/}"
             file_extension="${file##*.}"
             out_path="out/${file_path}/"
-            
+            root_path=$(echo "$file_path" | cut -d "/" -f1)
+
             echo "file_path: ${file_path}"
             echo "file_name: ${file_name}"
             echo "file_extension: ${file_extension}"
             echo "out_path: ${out_path}"
+            echo "root_path: ${root_path}"
 
-            if [[ $file_extension == "csv" ]]; then
+            if [[ ($file_extension == "csv" || $file_extension == "json") && $root_path != "out" ]]; then
```

Comment:
Might be worth put a comment here explaining why we're ignoring the `out` folder.